src/main/resources/init_db.sql

```diff
@@ -0,0 +1,12 @@
+create table manufacturers
```

Comment:
Lookup the CREATE Script in pgAdmin. Also, you should add CREATE Script for database.

Reply:
Thanks